### PR TITLE
Series colours updated

### LINKED ISSUE #370: Series colour
All series should be the same colour: `@secondary-orange: #FF7F00;`

### PR DESCRIPTION
**Pull request by rjmcneill**
Closes #370 

Updated line, point, and area to `@secondary-orange`.
Also added some slight fade onto the area since it was a bit bright.

RSI, MACD, and Volume have not been updated, they should match the style correctly

### CODE REVIEW

File: src/assets/styles/variables.less [hunk 1]
```diff
@@ -45,7 +45,7 @@
 // Plot Area
 @stroke-width: 3px;
 @area-fill: fade(@tertiary-blue, 75%);
```
Comment (jleft) [inline src/assets/styles/variables.less]: Could we update this to: `@area-fill: fade(@series-colour, 75%);`, then reference this variable for `.area` in primary.less?

**Comment by jleft** [comment]
Looks good - please could we also update the colours for the OHLC series to orange? Both the up and down variants (arguably we should use different colours for each, but this way we're being consistent and it's still very easy to change later).

**Comment by rjmcneill** [comment]
Sure, no bother.
Should the moving average and bollingerbands be left the colour they are though?

**Comment by jleft** [comment]
> Should the moving average and bollingerbands be left the colour they are though?

No changes need to be made to ma or boll - they're correct :smile:

**Comment by rjmcneill** [comment]
@jleft the build failed on CI.
I think you said before that this happens sometimes and it just needs rerun?

**Comment by jleft** [comment]
> @jleft the build failed on CI.
> I think you said before that this happens sometimes and it just needs rerun?

Sorry about that. Yeah, it occasionally fails when it shouldn't. All looks good now - thanks!

**Pull request merged**